### PR TITLE
update user

### PR DESCRIPTION
**Pull request by eventdift**
update user details from list by double clicking to show editor

### CODE REVIEW

File: src/app/areas/areas.component.ts [hunk 1]
```diff
@@ -24,7 +24,8 @@ export class AreasComponent implements OnInit {
 
   add() {
     const area = new Area(this.areaService);
-    this.areaService.add(area);
```
Comment (tolisster) [inline src/app/areas/areas.component.ts]: Why you removed possibility to add Area?

File: src/app/user-item/user-item.component.html [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {{ user.name }}
+
 <button class="delete" title="delete user"
-        (click)="delete(user)">x</button>
+        (click)="delete(user)" >x</button>
```
Comment (tolisster) [inline src/app/user-item/user-item.component.html]: what changes this change?

File: src/app/users/users.component.scss [hunk 3]
```diff
@@ -1,3 +1,15 @@
 :host {
   width: 240px;
+  position: relative;
+  .editor{
+    background-color: #ffffff;
+    position: absolute;
+    top: 00px;
```
Comment (tolisster) [inline src/app/users/users.component.scss]: enough 0;

File: src/app/users/users.component.scss [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
+    left: 0px;
```
Comment (tolisster) [inline src/app/users/users.component.scss]: too

File: src/app/user-item/user-item.component.html (hunk 2 above)
Comment (ASSASSIN-GROUP) [reply src/app/user-item/user-item.component.html]: I think it is his fault. only adding space?

File: src/app/areas/areas.component.ts (hunk 1 above)
Comment (eventdift) [reply src/app/areas/areas.component.ts]: so sorry. on it

File: src/app/user-item/user-item.component.html (hunk 2 above)
Comment (eventdift) [reply src/app/user-item/user-item.component.html]: working on it

File: src/app/users/users.component.scss (hunk 3 above)
Comment (eventdift) [reply src/app/users/users.component.scss]: ok, working on it

File: src/app/users/users.component.scss (hunk 4 above)
Comment (eventdift) [reply src/app/users/users.component.scss]: working on it

File: src/app/areas/areas.component.ts (hunk 1 above)
Comment (ASSASSIN-GROUP) [reply src/app/areas/areas.component.ts]: is that working for you?
I think in that case, you can't add any area. I mean when you click the + button, adding area won't work for you.

**Review by tolisster — APPROVED** [review]
Good

**Pull request merged**